src/components/NcChip/NcChip.vue

```diff
@@ -162,8 +162,18 @@ const props = withDefaults(defineProps<{
 	variant: 'secondary',
 })
 
-const emit = defineEmits(['close'])
-const slots = useSlots()
+const emit = defineEmits<{
+	/**
+	 * Emitted when the close button is clicked
+	 */
+	close: []
+}>()
+
+const slots = defineSlots<{
+	actions(): unknown
+	default(): unknown
+	icon(): unknown
```

Comment:
```ts
import type { Slot } from 'vue'
```

+

```suggestion
	actions?: Slot
	default?: Slot
	icon?: Slot
```

Reply:
I guess, that they don't want to make it "too complex", as in 90% cases you don't need it.

And because they might add new level magic to check slot content like for specific nodes.